update-doi-request/src/test/java/no/unit/nva/doi/requests/handlers/ApiUpdateDoiRequestHandlerTest.java

```diff
@@ -0,0 +1,341 @@
+        final TestAppender appender = LogUtils.getTestingAppender(UpdateDoiRequestHandler.class);
+        Publication publication = insertPublicationWithoutDoiRequest(
+            Clock.fixed(mockOneHourBefore, ZoneId.systemDefault()));
+
+        var updateDoiRequest = new ApiUpdateDoiRequest();
+        updateDoiRequest.setPublicationId(publication.getIdentifier().toString());
+        updateDoiRequest.setDoiRequestStatus(DoiRequestStatus.APPROVED);
+
+        var requestContext = objectMapper.createObjectNode();
+        GatewayResponse<Problem> response = sendRequest(updateDoiRequest, requestContext);
+
+        final Problem details = response.getBodyObject(Problem.class);
+
+        assertThat(response.getStatusCode(), Is.is(IsEqual.equalTo(HttpStatus.SC_FORBIDDEN)));
+
+        assertThat(details.getDetail(), is(equalTo("Forbidden")));
+        assertThat(appender.getMessages(),
+            containsString("Missing from requestContext: /authorizer/claims/custom:feideId"));
+    }
+
+    private void assertThatProblemDetailsDoesNotRevealSensitiveInformation(Problem details,
+                                                                           String username,
+                                                                           String publicationOwner) {
+        String errorMessage = details.getDetail();
+        assertThat(errorMessage, not(containsString(username)));
+        assertThat(errorMessage, not(containsString(publicationOwner)));
+    }
+
+    private void assertThatLogsContainReasonForForbiddenMessage(TestAppender appender, Publication publication) {
+        String expectedErrorMessage = String.format(WRONG_OWNER_ERROR, INVALID_USERNAME, publication.getOwner());
+        assertThat(appender.getMessages(), containsString(expectedErrorMessage));
+    }
+
+    @Test
+    public void handleRequestSucessfullyUpdatesStatusWhenPublicationIdIsValid() throws IOException, NotFoundException {
+        var publication = insertPublicationWithDoiRequest(Clock.fixed(mockNow,
+                ZoneId.systemDefault()));
+
+
+        var updateRequest = new ApiUpdateDoiRequest();
+        updateRequest.setPublicationId(publication.getIdentifier().toString());
+        updateRequest.setDoiRequestStatus(DoiRequestStatus.APPROVED);
+
+
+        var output = outpuStream();
+
+        GatewayResponse<ApiUpdateDoiResponse> response = sendRequest(updateRequest, validUsername(publication));
+
+        assertThat(response.getStatusCode(), is(equalTo(HttpStatus.SC_ACCEPTED)));
+
+        DoiRequestSummary actualDoiRequestSummary = readPublicationDirectlyFromDynamo(publication.getIdentifier());
+        DoiRequestSummary expectedDoiRequestSummary = expectedDoiRequestSummary(publication);
+
+        assertThat(actualDoiRequestSummary, is(equalTo(expectedDoiRequestSummary)));
+
+        var responseEntity = response.getBodyObject(ApiUpdateDoiResponse.class);
+        assertThat(responseEntity, is(equalTo(ApiUpdateDoiResponse.newBuilder()
```

Comment:
better to refactor this to `var expected = geneateResponseContainingHostInformation()` so that it will be obvious what is the expectation of the of the actual item.  Personally I tend to name my tested object "actual" and the expected object "expected" . I think we should all have a common naming practice for these two objects because it will make it very easy to see which entities are being tested.